protocol/messenger.go

```diff
@@ -88,6 +88,8 @@ var communityAdvertiseIntervalSecond int64 = 60 * 60
 // messageCacheIntervalMs is how long we should keep processed messages in the cache, in ms
 var messageCacheIntervalMs uint64 = 1000 * 60 * 60 * 48
 
+var baseStatusUrl string = "https://status.app/"
```

Comment:
Should we maybe also support:
- `http://`
- `status.app/...` directly without url scheme
- upper/mixed case `https://STATUS.App/...`

?

Reply:
I think this one is the most up-to-date doc?
https://github.com/vacp2p/rfc/blob/43c07f5290b644d668d3d4e6701a14075ef2d67b/content/docs/rfcs/60/README.md

(from https://github.com/vacp2p/rfc/pull/600)